packages/react-compass/src/rich-text-editor/controls/Control/Control.tsx

```diff
@@ -7,7 +7,7 @@ interface Props extends StyledComponentProps {
   active: boolean
   children?: React.ReactNode
   className?: string
-  onClick?: (event: React.MouseEvent<HTMLButtonElement>) => void
+  onClick?: ((event: React.MouseEvent<HTMLButtonElement>) => void) | undefined
```

Comment:
Check for using undefined like this?

Reply:
Hi @fpt-duongtx, I fixed all your comments. Please take a look.